dev/Makefile

```
@@ -0,0 +1,33 @@
+SHELL = bash
+.SHELLFLAGS = -e -c
+.ONESHELL:
+
+ENV ?= dev
+K3S_CHANNEL ?= stable
+
+env.auto.tfvars:
+	@echo 'name           = "$(ENV)"' > "$@"
+	@echo 'hcloud_token   = "$(HCLOUD_TOKEN)"' >> "$@"
+	@echo 'k3s_channel    = "$(K3S_CHANNEL)"' >> "$@"
+	@echo 'robot_user     = "$(ROBOT_USER)"' >> "$@"
+	@echo 'robot_password = "$(ROBOT_PASSWORD)"' >> "$@"
+
```

```suggestion
	@echo 'robot_enabled = "$(ROBOT_ENABLED >> "$@"
	@echo 'robot_user     = "$(ROBOT_USER)"' >> "$@"
	@echo 'robot_password = "$(ROBOT_PASSWORD)"' >> "$@"

```

Formatting is broken in the suggestion, please adjust accordingly :3